lib/file_download.rb

```diff
@@ -22,17 +19,14 @@ def fetch
   private
 
   def download_file(url, redirect_limit = MAX_REDIRECTS)
```

Comment:
We probably don't need the `redirect_limit` parameter now, as we're only calling this method once